content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+{{< /text >}}
+
+## Create remote cluster's kubeconfig for Istio Pilot
+
+The `istio-remote` helm chart creates a service account with minimal access for use by Istio Pilot
+discovery.
+
+1.  Prepare environment variables for building the `kubeconfig` file for the service account `istio-multi`:
+
+    {{< text bash >}}
+    $ export WORK_DIR=$(pwd)
+    $ CLUSTER_NAME=$(kubectl config view --minify=true -o "jsonpath={.clusters[].name}")
+    $ CLUSTER_NAME="${CLUSTER_NAME##*_}"
+    $ export KUBECFG_FILE=${WORK_DIR}/${CLUSTER_NAME}
+    $ SERVER=$(kubectl config view --minify=true -o "jsonpath={.clusters[].cluster.server}")
+    $ NAMESPACE=istio-system
+    $ SERVICE_ACCOUNT=istio-multi
+    $ SECRET_NAME=$(kubectl get sa ${SERVICE_ACCOUNT} -n ${NAMESPACE} -o jsonpath='{.secrets[].name}')
+    $ CA_DATA=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['ca\.crt']}")
+    $ TOKEN=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['token']}" | base64 --decode)
+    {{< /text >}}
+
+    __NOTE__: An alternative to `base64 --decode` is `openssl enc -d -base64 -A` on many systems.
+
+1.  Create a `kubeconfig` file in the working directory for the service account `istio-multi`:
+
+    {{< text bash >}}
+    $ cat <<EOF > ${KUBECFG_FILE}
+    apiVersion: v1
+    clusters:
+       - cluster:
+           certificate-authority-data: ${CA_DATA}
+           server: ${SERVER}
+         name: ${CLUSTER_NAME}
+    contexts:
+       - context:
+           cluster: ${CLUSTER_NAME}
+           user: ${CLUSTER_NAME}
+         name: ${CLUSTER_NAME}
+    current-context: ${CLUSTER_NAME}
+    kind: Config
+    preferences: {}
+    users:
+       - name: ${CLUSTER_NAME}
+         user:
+           token: ${TOKEN}
+    EOF
+    {{< /text >}}
+
+At this point, the remote clusters' `kubeconfig` files have been created in the `${WORK_DIR}` directory.
+The filename for a cluster is the same as the original `kubeconfig` cluster name.
+
+## Configure Istio control plane to discover the remote cluster
+
+Create a secret and label it properly for each remote cluster:
+
+{{< text bash >}}
+$ kubectl create secret generic ${CLUSTER_NAME} --from-file ${KUBECFG_FILE} -n ${NAMESPACE}
+$ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
+{{< /text >}}
+
+## Deploy Bookinfo Example Across Clusters
+
+1.  Install bookinfo on the first cluster.  Remove the `reviews-v3` deployment to deploy on remote:
+
+    {{< text bash >}}
+    $ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+    $ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+    $ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+    $ kubectl delete deployment reviews-v3
+    {{< /text >}}
+
+1.  Install the `reviews-v3` deployment on the remote.  _Note:_ The `ratings` service definition is added
+    to the remote cluster because `reviews-v3` is a client of `ratings` and creating the service object
```

Comment:
Consider putting the note after the .yaml file. As it currently reads, you are referencing content on the note that the reader hasn't seen yet.